src/main/java/com/devonfw/ide/sonarqube/common/impl/DevonfwJavaProfile.java

```diff
@@ -108,6 +133,27 @@ public void define(Context context) {
     devonfwJava.done();
   }
 
+  private void updateRulesTags(String tag, String repoKey, String ruleKey) throws IOException {
+
+    String hostname = "localhost";
```

Comment:
here hostname and port are hardcoded. This will not work. As this PR is WIP, I assume you are aware of this.
You may need to resolve variables like `sonar.host.url` instead...

Reply:
Did this `updateRulesTags` feature sneak into this PR accidentally? It does not seem to belong to the same issue/story (#93).

Reply:
Yes, I'm aware of this, that's why this PR is in WIP. The method updateRulesTags belongs to this PR because I have to add a new 'Tag' to 's2063' rule as I talked with Lmar.

Reply:
OK, thanks for your feedback. So PR #103 is not entirely replacing this PR and on the long run this PR will add an additional valuable feature.

Reply:
Yes exactly. I've created PR #103 because I understood that I should divide this PR for the changes that might already be merged. The question is if we need those changes with API call on SonarQube just to add and display additional 'tag' in a rule. I've tried to find another way to add this 'tag', but I couldn't find any.